src/lib.rs

```diff
@@ -64,8 +74,26 @@ pub fn drop_table(table: &str) -> io::Result<()> {
     Ok(())
 }
 
+#[allow(unused_must_use)]
```

Comment:
Here is on OSX: https://cloud.githubusercontent.com/assets/9837366/16052138/0f8b321c-3220-11e6-8d02-fd47be97472a.png

Windows: https://cloud.githubusercontent.com/assets/9837366/16051953/39e326d8-321f-11e6-8d34-94bad0589c10.png

Slightly different